### ISSUE TITLE
Search by student ID failed in "PHYS 022"

### ISSUE THREAD

**Issue by hydai** [henryyang42/NTHU_Course#10]
![2015-05-27 12 55 20](https://cloud.githubusercontent.com/assets/2776756/7817949/17d9f96a-040b-11e5-8d58-fcb6e4dc4f03.png)

**Comment by hydai** [comment]
Failed between "PHYS102B A" and "PHYS102BA", "EECS102B" and "EECS102BA"...

**Comment by henryyang42** [comment]
``` py
from data_center.models import *
ds = Department.objects.all()
for d in ds:
    d.dept_name = d.dept_name.replace('B A', 'BA')
    d.dept_name = d.dept_name.replace('B B', 'BB')
    d.save()

```

Use this to do a hotfix.
Some exception should be handled by human.

**Comment by hydai** [comment]
I will checkout all department id.
By the way, dept_name for EECS looks like "EECS101B", since it has only one class.

**Comment by henryyang42** [comment]
Yes, it's strange. Maybe we should merge 2 same `EECS101B` manually.

**Comment by hydai** [comment]
Failed list (We generate "BA" but actually they have only a "B")
- UPMT100B
- SCI 103B
- QF 100B
- LSIP103B
- LS 100B
- IPNS100B
- IPE 103B
- IEEM100B
- FL 100B
- EECS103B
- CL 100B
- CHEM100B
- CHE 103B
- BMES101B

**Comment by hydai** [comment]
I am wrong.
Our dept_name is correct, but it return more than an object.
![2015-05-27 10 31 29](https://cloud.githubusercontent.com/assets/2776756/7827582/b25acaa0-045b-11e5-91a8-699f6a4d279d.png)

**Comment by henryyang42** [comment]
I have told you we should merge 2 same EECS101B manually. 
`EECS` is one exception.

**Comment by henryyang42** [comment]
I will solve it on crawling.

**Comment by hydai** [comment]
Fix in 9d71f0ee27e1cacc10a3f803f3f3011b8cda15af, close.
